launcher_go/v2/config/config.go

```diff
@@ -0,0 +1,209 @@
+package config
+
+import (
+	"dario.cat/mergo"
+	"errors"
+	"fmt"
+	"github.com/discourse/discourse_docker/launcher_go/v2/utils"
+	"os"
+	"regexp"
+	"runtime"
+	"slices"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+var DefaultBootCommand = "/sbin/boot"
```

Comment:
```suggestion
const DefaultBootCommand = "/sbin/boot"
```

This will not change.